src/FieldLabel.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Deiucanta\Smart;
+
+trait FieldLabel
```

Comment:
I'd place this method inside the base class. Other than this, this is good to merge.

Reply:
Done